scanner/image/scanner/download-mappings.sh

```diff
@@ -20,11 +25,29 @@ urls=(
 for url in "${urls[@]}"; do
     filename=$(basename "$url")
     echo "Downloading ${url} > ${output_dir}/$filename"
-    curl --retry 3 -sSL --fail -o "${output_dir}/$filename" "$url"
+    curl --location --silent --fail --show-error --retry 3 --max-time 60 \
+        --output "${output_dir}/$filename" "$url"
     if [[ ! (-s "${output_dir}/$filename") ]]; then
         echo "${output_dir}/$filename is empty"
         exit 1
     fi
+
+    if command -v python &>/dev/null; then
```

Comment:
So I realized this is run from a few different places:

1. scanner-mapping-update: This uses the `ubuntu-latest` GitHub runner. From what I can tell, it has `python` - https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md
2. Konflux: I have no idea what the tekton pipeline runs on, but I don't think it has `python`. Perhaps it has `python3`? The Scanner CI image also doesn't have `python`, but rather it has `python3`, so perhaps more reason to potentially include `python3` in this. To see tekton pipeline output - https://console.redhat.com/application-pipeline/workspaces/rh-acs/applications/acs/pipelineruns/scanner-v4-on-push-xtbd2/logs?task=fetch-scanner-vuln-mappings
3. Within the upstream Dockerfile: doesn't seem to have either `python` nor `python3` (nor even `jq`) - registry.access.redhat.com/ubi8:latest
4. midstream: not a clue what it has

This may turn out to be more effort than we may have time for... I'm thinking we just ensure we use the correct links and ensure the file is not empty for the immediate solution and then we work on better validation later